spec/parsers/moov_parser_spec.rb

```diff
@@ -34,17 +34,24 @@ def deep_print_atoms(atoms, output, swimlanes = [])
       result = subject.call(File.open(moov_path, 'rb'))
 
       expect(result).not_to be_nil
-      expect(result.nature).to eq(:video)
-      expect(result.width_px).to be > 0
-      expect(result.height_px).to be > 0
+      expect(result.nature).to eq(:video) unless result.format == :m4a
```

Comment:
If the test is generic ("for al MOOVs")  we should also have generic assertions. For example we can verify that `#nature` is "one of :video or :audio". But I would strongly discourage adding conditionals in tests - if we need those, we should have separate tests for specific movie files and separate tests for specific audio files.

Reply:
Ahh, makes sense. How would you suggest handling it for the width_px and height_px fields?

Reply:
As soon as you have tests for each mov file, remove them in this spec :-)